src/redaction-properties/RedactionPropertiesManager.cpp

```diff
@@ -1,145 +1,162 @@
-#include <string>
-#include <vector>
+#include "../../include/redaction-properties/RedactionPropertiesManager.hpp"
+
 #include <iostream>
 #include <fstream>
 
 /**
- * This class is meant to load in a number of specified fields to redact from the partII section of BSM messages. This is used in the BSM Filter.
+ * @brief Construct a new Redaction Properties Manager object with a default path. Upon instantiation, fields are loaded from a file.
+ * 
  */
-class RedactionPropertiesManager {
-
-    public:
+RedactionPropertiesManager::RedactionPropertiesManager() {
+    std::string rpmDebug = getEnvironmentVariable("RPM_DEBUG");
+    if (rpmDebug == "true") {
```

Comment:
Do we need to handle casing issues here as well? Maybe taking advantage of std::isstringstream? ex: https://www.techiedelight.com/convert-string-bool-cpp/

Reply:
The RPM_DEBUG environment variable has been made case-insensitive and it will also accept "1" now.